packages/agoric-cli/test/bundle-cosmjs-math.test.js

```diff
@@ -0,0 +1,18 @@
+// A test to verify that importing @cosmsjs/math into an ESM via bundleSource
+// can see the exported Decimal constructor, which was at time of writing obscured
+// by https://github.com/endojs/endo/pull/2330, temporarily mitigated by a patch.
+// Placed here only because Agoric CLI depends on both bundleSource and CosmJS math.
```

Comment:
seems like it should go in cosmic-proto.

Reply:
done

Reply:
Looks like it was copied rather than moved.

I trust you'll get rid of the copy in agoric-cli before merging.

Reply:
Thank you for spotting. This wasn’t obvious locally.